jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -419,10 +419,8 @@
     builders:
       - update-automation
       - shell: |
-          if [[ $github_pr ]] ; then
```

Comment:
maybe instead of removing the if and putting it inside the .prep script simply add a 

rm -rvf $WORKSPACE/automation-git 

in there? and then we can remove the fancy rsync empty ?

Reply:
Then what if other things are left in the workspace? We could go with what @aojea suggested: use the workspace cleanup plugin, if you really want.

Reply:
Does it matter if other things are left in the workspace? Dirk's proposal only gets rid of the automation repo working copy, not of the whole workspace. Also, `rsync` is not really any safer than `rm -rf` - only more obscure, so +1 for removing it...

Regarding the workspace cleanup plugin: let's not introduce a dependency on a Jenkins plugin for so trivial a task. Shell scripts are way easier to debug if they go wrong.

Reply:
I replaced the rsync with rm.